modules/Rubrics/src/Visualise.php

```diff
@@ -155,6 +158,13 @@ public function renderVisualise($legend = true, $image = false, $path = '', $id
                         this.options.animation.onComplete = null;
                     }'),
                 ];
+            } else {
```

Comment:
Interested to know why the commented section has been left in here. I know there will be a good reason!

Reply:
Oops! Nope, no good reason this time 😆